tests/format/format_test.py

```diff
@@ -23,6 +23,88 @@ def test_variable_default(var):
     assert '{:}'.format(var) == str(var)
 
 
+@pytest.mark.parametrize('s', ('^', ''))
+def test_variable_default_length_central(s):
+    var = sc.arange('x', 10)
```

Comment:
What happens for 2-D or higher? Does it raise?

Reply:
No, it flattens the array (same as `__str__`):
```python
In [3]: f'{sc.array(dims=["x", "y"], values=[[1, 2, 3], [4, 5, 6]]):<}'
Out[3]: '<scipp.Variable> (x: 2, y: 3)      int64  [dimensionless]  [1, 2, 3, 4, ...]'
```